keyboards/prototypist/pt80/info.json

```diff
@@ -0,0 +1,123 @@
+{
+    "manufacturer": "Proto[Typist]",
+    "keyboard_name": "PT-80",
+    "maintainer": "Anjheos",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B7", "B6", "B5", "B4", "B3", "B0", "A7", "B1", "B2", "B10", "A6", "A5", "A4", "A3", "C13", "A2", "C14"],
+        "rows": ["B14", "A8", "A9", "B13", "B11", "B12"]
+    },
+    "processor": "STM32F303",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0x4A46"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
Same here
```suggestion
        "LAYOUT": {
```

Reply:
added ISO and ANSI in info.json (for both) giving reason for LAYOUT_all.